incompatible path on baseline for phpstan (alone) and grumphp.

**Describe the bug**
`composer fop_check` (grumphp) and `vendor/bin/phpstan` handle the .devtools/phpstan_baseline.neon with different path root.

So if an error on phpstan_baseline.neon is reported with a correct path for phpstan standalone, it's ok for grumphp.
The opposite it also true.

```neon
parameters:
	ignoreErrors:
		-
			message: "#^Call to an undefined method Symfony\\\\Component\\\\Console\\\\Application\\:\\:getKernel\\(\\)\\.$#"
			count: 1
			path: src/Commands/CheckServiceContainer.php

# That's fine, dump() can handle multiple values - This is not an error. https://github.com/symfony/var-dumper/blob/3.4/Resources/functions/dump.php
		-
			message: "#^Function dump invoked with 2 parameters, 1 required\\.$#"
			count: 1
			path: ../src/Commands/Configuration/Export.php
```

with this baseline, first path is ok for grumphp, not phpstan directly.
second path it not ok for grumphp but ok for phpstan.

Maybe we could move the baseline file at the root.

No one reported this recent problem.
Do you use grumphp on pre-commit hook (or `composer fop_check`) ? @okom3pom @Kaudaj ?

**With composer fop_check**

okom3pom@okom3pom-N501VW:~/www/178/PrestaShop/modules/fop_console$ composer fop_check
> grumphp run --ansi
GrumPHP is sniffing your code!

Running tasks with priority 0!
==============================

Running task 1/2: phpcsfixer... ✔
Running task 2/2: phpstan... ✘

             ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
           ▄▄▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▌
         ▐▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▄
        ▐▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▌
       ▐▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▌
  ▄███▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▌
 █▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▌
 ▐█▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▌
   ▀█▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▌
     ▀▀▓▓▓▓▓▓▓▓▓▓▓▓█▀▀▀▀▀▀▀▀▀▀▀▀▀▀████████████▄
      ▄███████                       ██████████
     ███████▀  ▀▀▀▀▀▄      ▄▀▀▀▀▀     █████ ▀
      ▐████      ▐██        ▐██        ████▌
      ████▌                            ███
       ▌██▌           ▄▄ ▄▄           ▐███
        ███       ▄▄▄▄▄▄▄▄▄▄▄▄       ▐███
         ██▄ ▐███████████████████████████
        █▀███████████▀     ▀▀███████████
          ██████████▄███████▄███████████
         ▐█████████████████████████████
          █████████████████████████████
           ██ █████████████████████▐██▀
            ▀ ▐███████████████████▌ ▐▀
                ████▀████████▀▐███
                 ▀█▌  ▐█████  ██▌
                        ██▀   ▐▀

       ██████████████████████████████████
       █░░░░░░▀█▀░░░░░░▀█░░░░░░▀█▀░░░░░▀█
       █░░▐█▌░░█░░░██░░░█░░██░░░█░░░██░░█
       █░░▐█▌░░█░░░██░░░█░░██░░░█░░░██░░█
       █░░▐█▌░░█░░░██░░░█░░░░░░▄█░░▄▄▄▄▄█
       █░░▐█▌░░█░░░██░░░█░░░░████░░░░░░░█
       █░░░█░░░█▄░░░░░░▄█░░░░████▄░░░░░▄█
       ██████████████████████████████████

phpstan
=======

Phpstan configuration : _PS_ROOT_DIR_ successfully set to /home/okom3pom/www/178/PrestaShop/
Warning: Use of undefined constant _THEME_NAME_ - assumed '_THEME_NAME_' (this will throw an Error in a future version of PHP) in /var/www/html/178/PrestaShop/config/defines_uri.inc.php on line 29

Warning: Use of undefined constant __PS_BASE_URI__ - assumed '__PS_BASE_URI__' (this will throw an Error in a future version of PHP) in /var/www/html/178/PrestaShop/config/defines_uri.inc.php on line 30

Warning: Use of undefined constant _THEME_NAME_ - assumed '_THEME_NAME_' (this will throw an Error in a future version of PHP) in /var/www/html/178/PrestaShop/config/defines_uri.inc.php on line 30

Warning: Use of undefined constant __PS_BASE_URI__ - assumed '__PS_BASE_URI__' (this will throw an Error in a future version of PHP) in /var/www/html/178/PrestaShop/config/defines_uri.inc.php on line 39

Warning: Use of undefined constant _THEME_NAME_ - assumed '_THEME_NAME_' (this will throw an Error in a future version of PHP) in /var/www/html/178/PrestaShop/config/defines_uri.inc.php on line 40

Warning: Use of undefined constant __PS_BASE_URI__ - assumed '__PS_BASE_URI__' (this will throw an Error in a future version of PHP) in /var/www/html/178/PrestaShop/config/defines_uri.inc.php on line 46

Warning: Use of undefined constant __PS_BASE_URI__ - assumed '__PS_BASE_URI__' (this will throw an Error in a future version of PHP) in /var/www/html/178/PrestaShop/config/defines_uri.inc.php on line 63

Warning: Use of undefined constant __PS_BASE_URI__ - assumed '__PS_BASE_URI__' (this will throw an Error in a future version of PHP) in /var/www/html/178/PrestaShop/config/defines_uri.inc.php on line 64

Warning: Use of undefined constant __PS_BASE_URI__ - assumed '__PS_BASE_URI__' (this will throw an Error in a future version of PHP) in /var/www/html/178/PrestaShop/config/defines_uri.inc.php on line 65

Warning: Use of undefined constant __PS_BASE_URI__ - assumed '__PS_BASE_URI__' (this will throw an Error in a future version of PHP) in /var/www/html/178/PrestaShop/config/defines_uri.inc.php on line 66

Warning: Use of undefined constant __PS_BASE_URI__ - assumed '__PS_BASE_URI__' (this will throw an Error in a future version of PHP) in /var/www/html/178/PrestaShop/config/defines_uri.inc.php on line 67
Detected PS version 1.7.8.0
 ------ ----------------------------------------------------------------------- 
  Line   src/Commands/CheckServiceContainer.php                                 
 ------ ----------------------------------------------------------------------- 
         Ignored error pattern #^Call to an undefined method                    
         Symfony\\Component\\Console\\Application\:\:getKernel\(\)\.$# in path  
         /var/www/html/178/PrestaShop/modules/fop_console/src/Commands/CheckSe  
         rviceContainer.php was not matched in reported errors.                 
 ------ ----------------------------------------------------------------------- 

 ------ ----------------------------------------------------------------------- 
  Line   src/Commands/Configuration/Export.php                                  
 ------ ----------------------------------------------------------------------- 
         Ignored error pattern #^Function dump invoked with 2 parameters, 1     
         required\.$# in path                                                   
         /var/www/html/178/PrestaShop/modules/fop_console/src/Commands/Configu  
         ration/Export.php was not matched in reported errors.                  
 ------ ----------------------------------------------------------------------- 

 [ERROR] Found 2 errors                                                         


Note: Using configuration file /var/www/html/178/PrestaShop/modules/fop_console/phpstan.neon.dist.
PHP Warning:  Use of undefined constant _THEME_NAME_ - assumed '_THEME_NAME_' (this will throw an Error in a future version of PHP) in /var/www/html/178/PrestaShop/config/defines_uri.inc.php on line 29
PHP Warning:  Use of undefined constant __PS_BASE_URI__ - assumed '__PS_BASE_URI__' (this will throw an Error in a future version of PHP) in /var/www/html/178/PrestaShop/config/defines_uri.inc.php on line 30
PHP Warning:  Use of undefined constant _THEME_NAME_ - assumed '_THEME_NAME_' (this will throw an Error in a future version of PHP) in /var/www/html/178/PrestaShop/config/defines_uri.inc.php on line 30
PHP Warning:  Use of undefined constant __PS_BASE_URI__ - assumed '__PS_BASE_URI__' (this will throw an Error in a future version of PHP) in /var/www/html/178/PrestaShop/config/defines_uri.inc.php on line 39
PHP Warning:  Use of undefined constant _THEME_NAME_ - assumed '_THEME_NAME_' (this will throw an Error in a future version of PHP) in /var/www/html/178/PrestaShop/config/defines_uri.inc.php on line 40
PHP Warning:  Use of undefined constant __PS_BASE_URI__ - assumed '__PS_BASE_URI__' (this will throw an Error in a future version of PHP) in /var/www/html/178/PrestaShop/config/defines_uri.inc.php on line 46
PHP Warning:  Use of undefined constant __PS_BASE_URI__ - assumed '__PS_BASE_URI__' (this will throw an Error in a future version of PHP) in /var/www/html/178/PrestaShop/config/defines_uri.inc.php on line 63
PHP Warning:  Use of undefined constant __PS_BASE_URI__ - assumed '__PS_BASE_URI__' (this will throw an Error in a future version of PHP) in /var/www/html/178/PrestaShop/config/defines_uri.inc.php on line 64
PHP Warning:  Use of undefined constant __PS_BASE_URI__ - assumed '__PS_BASE_URI__' (this will throw an Error in a future version of PHP) in /var/www/html/178/PrestaShop/config/defines_uri.inc.php on line 65
PHP Warning:  Use of undefined constant __PS_BASE_URI__ - assumed '__PS_BASE_URI__' (this will throw an Error in a future version of PHP) in /var/www/html/178/PrestaShop/config/defines_uri.inc.php on line 66
PHP Warning:  Use of undefined constant __PS_BASE_URI__ - assumed '__PS_BASE_URI__' (this will throw an Error in a future version of PHP) in /var/www/html/178/PrestaShop/config/defines_uri.inc.php on line 67
To skip commit checks, add -n or --no-verify flag to commit command
Script grumphp run --ansi handling the fop_check event returned with error code 1

### `composer fop_check` on `dev` branch :

> grumphp run --ansi
GrumPHP is sniffing your code!

Running tasks with priority 0!
==============================

Running task 1/2: phpcsfixer... ✔
Running task 2/2: phpstan... ✘
             ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
           ▄▄▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▌
         ▐▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▄
        ▐▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▌
       ▐▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▌
  ▄███▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▌
 █▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▌
 ▐█▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▌
   ▀█▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▌
     ▀▀▓▓▓▓▓▓▓▓▓▓▓▓█▀▀▀▀▀▀▀▀▀▀▀▀▀▀████████████▄
      ▄███████                       ██████████
     ███████▀  ▀▀▀▀▀▄      ▄▀▀▀▀▀     █████ ▀
      ▐████      ▐██        ▐██        ████▌
      ████▌                            ███
       ▌██▌           ▄▄ ▄▄           ▐███
        ███       ▄▄▄▄▄▄▄▄▄▄▄▄       ▐███
         ██▄ ▐███████████████████████████
        █▀███████████▀     ▀▀███████████
          ██████████▄███████▄███████████
         ▐█████████████████████████████
          █████████████████████████████
           ██ █████████████████████▐██▀
            ▀ ▐███████████████████▌ ▐▀
                ████▀████████▀▐███
                 ▀█▌  ▐█████  ██▌
                        ██▀   ▐▀

       ██████████████████████████████████
       █░░░░░░▀█▀░░░░░░▀█░░░░░░▀█▀░░░░░▀█
       █░░▐█▌░░█░░░██░░░█░░██░░░█░░░██░░█
       █░░▐█▌░░█░░░██░░░█░░██░░░█░░░██░░█
       █░░▐█▌░░█░░░██░░░█░░░░░░▄█░░▄▄▄▄▄█
       █░░▐█▌░░█░░░██░░░█░░░░████░░░░░░░█
       █░░░█░░░█▄░░░░░░▄█░░░░████▄░░░░░▄█
       ██████████████████████████████████


phpstan
=======

Phpstan configuration : _PS_ROOT_DIR_ successfully set to /var/www/prestashop-sandboxDetected PS version 1.7.8.0
 ------ -----------------------------------------------------------------------
  Line   src/Commands/CheckServiceContainer.php
 ------ -----------------------------------------------------------------------
         Ignored error pattern #^Call to an undefined method
         Symfony\\Component\\Console\\Application\:\:getKernel\(\)\.$# in path
         /var/www/prestashop-sandbox/modules/fop_console/src/Commands/CheckSer
         viceContainer.php was not matched in reported errors.
 ------ -----------------------------------------------------------------------

 ------ -----------------------------------------------------------------------
  Line   src/Commands/Configuration/Export.php
 ------ -----------------------------------------------------------------------
         Ignored error pattern #^Function dump invoked with 2 parameters, 1
         required\.$# in path
         /var/www/prestashop-sandbox/modules/fop_console/src/Commands/Configur
         ation/Export.php was not matched in reported errors.
 ------ -----------------------------------------------------------------------

 [ERROR] Found 2 errors


Note: Using configuration file /var/www/prestashop-sandbox/modules/fop_console/phpstan.neon.dist.
To skip commit checks, add -n or --no-verify flag to commit command
Script grumphp run --ansi handling the fop_check event returned with error code 1

### Pre-commit hook :

Running `git -c user.useConfigOnly=true commit --quiet --allow-empty-message --file -` gives me this :

hint: The '.git/hooks/pre-commit' hook was ignored because it's not set as executable.
hint: You can disable this warning with `git config advice.ignoredHook false`.
(reading log message from standard input)

Is it the right way to test pre-commit hook? If not, could you give me the right way to do it?

--- a/.github/workflows/php-cs-fixer.yml
+++ b/.github/workflows/php-cs-fixer.yml
@@ -30,4 +30,4 @@ jobs:
         run: |
           cd ${{ github.workspace }}
           ./vendor/bin/php-cs-fixer --version
-          ./vendor/bin/php-cs-fixer fix --dry-run --diff ./
+          ./vendor/bin/php-cs-fixer fix --config=".php_cs-fixer.dist.php" --dry-run --diff ./

--- a/.php_cs-fixer.dist.php
+++ b/.php_cs-fixer.dist.php
@@ -19,6 +19,7 @@ $fopPsConfig = new class extends PrestaShop\CodingStandards\CsFixer\Config {
 
 $fopPsConfig
     ->setUsingCache(true)
+    ->setCacheFile('.php_cs.cache')
     ->getFinder()
     ->in(__DIR__)
     ->exclude('vendor')

--- a/composer.json
+++ b/composer.json
@@ -37,7 +37,7 @@
   "require-dev": {
     "prestashop/php-dev-tools": "4.*",
     "phpro/grumphp": "0.19.1",
-    "phpstan/phpstan": "^0.12.58",
+    "phpstan/phpstan": "1.*",
     "phpunit/phpunit": "^7.0"
   },
   "scripts": {

--- a/grumphp.yml.dist
+++ b/grumphp.yml.dist
@@ -3,7 +3,7 @@ grumphp:
     phpcsfixer:
       config: .php_cs-fixer.dist.php
     phpstan:
-      level: 5 # Needed because Grum overwrite level at 0 by default in 0.19.1 version
+      level: 5 # Needed because Grumphp overwrites level at 0, by default in 0.19.1 version
       ignore_patterns: ['/(^\.|\/\.)/'] # Needed because PHPStan ignore dotfiles
       force_patterns: [] # Add dotfile to not ignore here
   parallel:

--- a/grumphp.yml.dist
+++ b/grumphp.yml.dist
@@ -3,8 +3,9 @@ grumphp:
     phpcsfixer:
       config: .php_cs-fixer.dist.php
     phpstan:
-      level: 5 # Otherwise, Grum overwrite level at 0 by default in 0.19.1 version
-      ignore_patterns: ['/(^\.|\/\.)/'] # Ignore dot files because PHPStan does
+      level: 5 # Needed because Grum overwrite level at 0 by default in 0.19.1 version
+      ignore_patterns: ['/(^\.|\/\.)/'] # Needed because PHPStan ignore dotfiles
+      force_patterns: [] # Add dotfile to not ignore here
   parallel:
     enabled: false
   fixer:

--- a/grumphp.yml.dist
+++ b/grumphp.yml.dist
@@ -2,7 +2,9 @@ grumphp:
   tasks:
     phpcsfixer:
       config: .php_cs-fixer.dist.php
-    phpstan: ~
+    phpstan:
+      level: 5 # Otherwise, Grum overwrite level at 0 by default in 0.19.1 version
+      ignore_patterns: ['/(^\.|\/\.)/'] # Ignore dot files because PHPStan does
   parallel:
     enabled: false
   fixer:

--- a/phpstan.neon.dist
+++ b/phpstan.neon.dist
@@ -14,6 +14,6 @@ parameters:
 
 includes:
   - %currentWorkingDirectory%/.devtools/.phpstan_bootstrap.neon
-  - %currentWorkingDirectory%/.devtools/phpstan_baseline.neon
+  - %currentWorkingDirectory%/phpstan_baseline.neon
   - %currentWorkingDirectory%/vendor/prestashop/php-dev-tools/phpstan/ps-module-extension.neon
   - %currentWorkingDirectory%/.devtools/fop_names_rules.neon

--- a/phpstan.neon.dist
+++ b/phpstan.neon.dist
@@ -7,7 +7,7 @@ parameters:
     - .devtools/phpstan/stubs/
     - tests/Resources/
   reportUnmatchedIgnoredErrors: true
-  level: 5 # If changed, need to be changed in grumphp.yml.dist too
+  level: 5 # If changed, needs to be changed in grumphp.yml.dist too
   stubFiles:
     - .devtools/phpstan/stubs/Category.stub
     - .devtools/phpstan/stubs/Product.stub

--- a/phpstan.neon.dist
+++ b/phpstan.neon.dist
@@ -1,7 +1,7 @@
 parameters:
   paths:
     - ./
-  excludes_analyse:
+  excludePaths:
     - vendor/
     - .devtools/phpstan/stubs/
     - tests/Resources/

--- a/phpstan.neon.dist
+++ b/phpstan.neon.dist
@@ -1,16 +1,16 @@
 parameters:
   paths:
+    # If dotfile path added, need to be added in grumphp.yml.dist too
     - ./
   excludePaths:
     - vendor/
     - .devtools/phpstan/stubs/
     - tests/Resources/
   reportUnmatchedIgnoredErrors: true
-  level: 5
+  level: 5 # If changed, need to be changed in grumphp.yml.dist too
   stubFiles:
-      - .devtools/phpstan/stubs/Category.stub
-      - .devtools/phpstan/stubs/Product.stub
-
+    - .devtools/phpstan/stubs/Category.stub
+    - .devtools/phpstan/stubs/Product.stub
 
 includes:
   - %currentWorkingDirectory%/.devtools/.phpstan_bootstrap.neon

--- a/phpstan_baseline.neon
+++ b/phpstan_baseline.neon
@@ -3,22 +3,22 @@ parameters:
 		-
 			message: "#^Call to an undefined method Symfony\\\\Component\\\\Console\\\\Application\\:\\:getKernel\\(\\)\\.$#"
 			count: 1
-			path: ../src/Commands/Container/ContainerCheck.php
+			path: src/Commands/Container/ContainerCheck.php
 
 # That's fine, dump() can handle multiple values - This is not an error. https://github.com/symfony/var-dumper/blob/3.4/Resources/functions/dump.php
 		-
 			message: "#^Function dump invoked with 2 parameters, 1 required\\.$#"
 			count: 1
-			path: ../src/Commands/Configuration/ConfigurationExport.php
+			path: src/Commands/Configuration/ConfigurationExport.php
 
 # The rule can not determine if the command name is correct because it's constructed at runtime.
 		-
 			message: "#^Console command name can not be extracted\\. Therefore consistency with classname can't be checked\\.$#"
 			count: 1
-			path: ../src/Commands/Image/ImageGenerateAbstract.php
+			path: src/Commands/Image/ImageGenerateAbstract.php
 
 #	This error happens in a stub's test file. It must be ignored in the baseline otherwise the phpstan's unit test will fail.
 		-
 			message: "#^Call to an undefined method FOP\\\\Console\\\\Tests\\\\Resources\\\\Commands\\\\Domain\\\\DomainAction\\:\\:setName\\(\\)\\.$#"
 			count: 1
-			path: ../tests/Resources/Commands/Domain/DomainAction.php
+			path: tests/Resources/Commands/Domain/DomainAction.php

--- a/src/Commands/Container/ContainerCheck.php
+++ b/src/Commands/Container/ContainerCheck.php
@@ -22,7 +22,6 @@ namespace FOP\Console\Commands\Container;
 
 use Exception;
 use FOP\Console\Command;
-use PrestaShopBundle\Exception\NotImplementedException;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
@@ -67,8 +66,6 @@ final class ContainerCheck extends Command
             ) {
                 try {
                     $this->getContainer()->get($serviceId);
-                } catch (NotImplementedException $notImplementedException) {
-                    $services[] = [$serviceId, '<comment>' . $this->formatException($notImplementedException) . '</comment>'];
                 } catch (Exception $exception) {
                     $services[] = [$serviceId, '<fg=red>' . $this->formatException($exception) . '</>'];
                 } catch (Throwable $error) {

--- a/src/Commands/Employee/EmployeeChangePassword.php
+++ b/src/Commands/Employee/EmployeeChangePassword.php
@@ -28,7 +28,6 @@ use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Console\Question\Question;
-use Symfony\Component\Console\Style\SymfonyStyle;
 use Tools;
 use Validate;
 

--- a/src/Commands/Module/ModuleRename.php
+++ b/src/Commands/Module/ModuleRename.php
@@ -330,7 +330,7 @@ final class ModuleRename extends Command
             ],
         ];
 
-        if (isset($this->newModuleInfos['author'])) {
+        if (!empty($this->newModuleInfos['author'])) {
             array_push(
                 $searchAndReplacePairs,
                 [
@@ -381,15 +381,11 @@ final class ModuleRename extends Command
         $replacePairs = [];
 
         foreach ($searchAndReplacePairs as $searchAndReplacePair) {
-            if (!isset($searchAndReplacePair['search'])
-                || empty($searchAndReplacePair['search'])) {
+            if (empty($searchAndReplacePair['search'])) {
                 continue;
             }
             $search = $searchAndReplacePair['search'];
-
-            $replace = isset($searchAndReplacePair['replace'])
-                ? $searchAndReplacePair['replace']
-                : '';
+            $replace = $searchAndReplacePair['replace'];
 
             $caseFormats = isset($searchAndReplacePair['caseFormats'])
                 ? $searchAndReplacePair['caseFormats']

--- a/tests/Unit/PhpStanNamesConsistencyRuleTest.php
+++ b/tests/Unit/PhpStanNamesConsistencyRuleTest.php
@@ -53,7 +53,7 @@ class PhpStanNamesConsistencyRuleTest extends RuleTestCase
         // this is because PhpStanNamesConsistencyRule::nodeIsInClassFopCommand() checks the FQDN and therefore it needs to be changed
         // with the namespace of the tested class (FOP\Console\Tests\Resources\Commands\Domain).
         /* @phpstan-ignore-next-line */
-        return new class ($mockedValidatorService) extends PhpStanNamesConsistencyRule {
+        return new class($mockedValidatorService) extends PhpStanNamesConsistencyRule {
             public const FOP_BASE_COMMAND_CLASS_NAME = 'FOP\Console\Tests\Resources\Commands\Command';
         };
     }

--- a/tests/Validator/FOPCommandFormatsValidator.php
+++ b/tests/Validator/FOPCommandFormatsValidator.php
@@ -31,7 +31,6 @@ use Exception; /**
  * - command name (symfony command name) is consistent with <Domain> and <Action>
  * - service name (symfony service declaration) is consistent with <Domain> and <Action>
  */
-
 class FOPCommandFormatsValidator
 {
     /**

